images/.htaccess

```diff
@@ -0,0 +1,6 @@
+<IfModule rewrite_module>
+	RewriteEngine On
+	RewriteOptions inherit
+	# Fix for bug T64289
+	Options +FollowSymLinks
+</IfModule>
```

Comment:
Are you sure custom .htaccess is necessary for images? Please note that the image is configured to copy over bundled contents of the `images` directory on initial setup https://github.com/CanastaWiki/Canasta/blob/master/_sources/scripts/run-apache.sh#L62 so fresh installs will get the one bundled with MediaWiki release